### PR TITLE
[Cherry-pick] Add documentation concern standard format

### PR DESCRIPTION
**Pull request by mickaelandrieu**
Hello,

this is a cherry-pick of https://github.com/akeneo/pim-docs/pull/410 into 1.6 branch, we also need it in `1.5` and `1.7` branches.

Regards

### CODE REVIEW

File: reference/standard_format/index.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
+Standard format
+===============
+
+The standard format is a normalized array representation of the objects of the PIM.
+It it used to manipulate (query/update), describe or even sometimes store these objects inside the PIM.
+Currently it is not designed to provide a representation of these objects outside the PIM.
+
+The standard format is consistent in term of:
```
Comment (MarieMinasyan) [inline reference/standard_format/index.rst]: _terms of_

File: reference/standard_format/products.rst [hunk 2]
```diff
@@ -0,0 +1,499 @@
+Products
+========
+
+Common structure
+----------------
+
+Product contain inner fields and product values that are linked to attributes. All products have the same fields (identifier, label, family, groups, variant groups, categories, associations, status, dates of creation and update) while product values are flexible among products.
```
Comment (MarieMinasyan) [inline reference/standard_format/products.rst]: _Products contain_ or _A product contains_

File: reference/standard_format/products.rst [hunk 3]
```diff
@@ -0,0 +1,499 @@
...
+|               |                |                                                                            |                                                              |
++---------------+----------------+----------------------------------------------------------------------------+--------------------------------------------------------------+
+| associations  | array          | ``["X_SELL" => ["groups" => [0 => "groupA"],"products" => [0 => "foo"]]]`` | | see below                                                  |
+|               |                |                                                                            |                                                              |
++---------------+----------------+----------------------------------------------------------------------------+--------------------------------------------------------------+
+
+Associations
+------------
+
+The structure of the array is composed as below:
```
Comment (MarieMinasyan) [inline reference/standard_format/products.rst]: _as follows_

File: reference/standard_format/products.rst [hunk 4]
```diff
@@ -0,0 +1,499 @@
...
+        "data" => "the data in English"
+      ]
+      1 => array:3 [
+        "locale" => "fr_FR"
+        "scope" => null
+        "data" => "la donnée en français"
+      ]
+    ]
+
+Depending on the type of the product value, the data key can have different structure:
```
Comment (MarieMinasyan) [inline reference/standard_format/products.rst]: _different structures_

**Comment by mickaelandrieu** [comment]
Closed as the "standard" format won't for 1.6 branches and older.

**Pull request closed without merging**